AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -70,12 +66,12 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
         requested_azel = katpoint.rad2deg(np.array(requested_azel))
 
    
-    gaussian_centre     = np.zeros((chunk_size* 2,2,len(h5.ants)) )
-    gaussian_centre_std = np.zeros((chunk_size* 2,2,len(h5.ants)) )
-    gaussian_width      = np.zeros((chunk_size* 2,2,len(h5.ants)) )
-    gaussian_width_std  = np.zeros((chunk_size* 2,2,len(h5.ants)) )
-    gaussian_height     = np.zeros((chunk_size* 2,len(h5.ants)) )
-    gaussian_height_std = np.zeros((chunk_size* 2,len(h5.ants)) )
+    gaussian_centre     = np.zeros((chunk_size* 2,2,len(h5.ants))) +np.nan
```

Comment:
Use an explicit constructor instead of doing arithmetic:
```
gaussian_centre = np.full((chunk_size * 2, 2, len(h5.ants)), np.nan)
```

Reply:
thanks - i've implemented your suggestion.